en/docs/Learn/DesignAPI/Endpoints/Resiliency/prevent-api-suspension.md

```diff
@@ -48,4 +45,4 @@ To avoid backend endpoint suspension,
     ```
 
 
-For more details on configuring different timeouts, see [Timeout configurations for an API call](https://docs.wso2.com/display/SHAN/Tuning+Performance#TuningPerformance-TimeoutconfigurationsforanAPIcall) in the Performance Tuning guide.
+For more details on configuring different timeouts, see [Timeout configurations for an API call](https://apim.docs.wso2.com/en/next/Administer/ProductAdministration/PerfromanceTuningAndTestResults/tuning-performance/) in the Performance Tuning guide.
```

Comment:
@nalakas the image paths are incorrect
- We should not be giving the doc path like this (https://apim.docs.wso2.com/en/next/Administer/ProductAdministration/PerfromanceTuningAndTestRe
sults/tuning-performance/) in the Performance Tuning guide.